content/08-cardano-sidechains/01-basics/01-introduction-sidechains.mdx

```diff
@@ -63,12 +63,8 @@ For a full description of the theoretical underpinning of the design, refer to t
 
 ## Closing remarks
 
-The sidechains toolkit is the latest delivery on the Cardano roadmap within Basho, the scaling phase, introducing higher programmability, interoperability, and scalability. Blockchain developers will be able to create custom sidechains more easily. Eventually, IOG hopes to see a family of Cardano sidechains and partner chains emerging.
+The sidechains toolkit is the latest delivery on the Cardano roadmap within Basho, the scaling phase, introducing higher programmability, interoperability, and scalability. Blockchain developers will be able to create custom sidechains more easily. Eventually, IOG hopes to see a family of Cardano sidechains and partner chains emerging.  
 
-The EVM sidechain public testnet will be released in January 2023 allowing the community to deploy DApps, create smart contracts, and move tokens between the testing environment chains.
+The toolkit released in February is not a complete solution. There are some known areas for improvement, like the bridge experience, SPO rewards mechanism and the security model. All these areas will be worked on with the community as we go.  
 
-The toolkit being released in January is not a complete solution. There are some known areas for improvement, like the bridge experience, SPO rewards mechanism and the security model. All these areas will be worked on with the community as we go.
```

Comment:
I think we can leave this text

> There are some known areas for improvement, like the bridge experience, SPO rewards mechanism and the security model. All these areas will be worked on with the community as we go.

Reply:
So do you want to delete the text 'The toolkit released in February is not a complete solution.'?

Reply:
@NeilBurgess42 yes

Reply:
OK